server/controllers/finance/priceList.js

```diff
@@ -321,3 +322,35 @@ exports.delete = function del(req, res, next) {
 function isEmptyObject(object) {
   return Object.keys(object).length === 0;
 }
+
+exports.download = (req, res, next) => {
```

Comment:
Most other controllers have a specialized controllers under `finance/reports`.  It makes it easier to read and ensure we don't have really long files.  Would it be possible to move this code under reports?